modules/hooks.nix

```
@@ -145,7 +145,7 @@ in
           output =
             mkOption {
               description = lib.mdDoc "Output format.";
-              types = types.nullOr types.enum [ "check" "list-diffrent" ];
+              type = types.nullOr types.enum [ "check" "list-diffrent" ];
```

```suggestion
              type = types.nullOr types.enum [ "check" "list-different" ];
```

There is another typo on this line in the `enum`, my bad.